server/config/config_test.go

```diff
@@ -0,0 +1,152 @@
+package config
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIsValid(t *testing.T) {
+	for _, testCase := range []struct {
+		description string
+		config      *Configuration
+		errMsg      string
+	}{
+		{
+			description: "valid configuration",
```

Comment:
```suggestion
			description: "valid configuration",
```

we can either write a proper statement or sentence like
```
with valid configuration
with invalid configuration where AzureDevopsAPIBaseURL is empty
```

or can write as sort description like 
```
configuration: valid,
configuration: empty AzureDevopsAPIBaseURL
```

fix for all such places